docs/requirements.txt

```diff
@@ -10,5 +10,6 @@ pydata-sphinx-theme~=0.14
 sphinx-autobuild
 sphinx-design
 sphinx-copybutton
+sphinx-lint
```

Comment:
Do we need this?
I think that `sphinx-lint` is installed by `pip install -e dev/archery[lint]`.

Reply:
We don't need it, good catch. Removed in b526a4d844b143b6ef5c414abf0366b274645817.